parsers/src/main/scala/uk/gov/ons/addressIndex/parsers/Features.scala

```diff
@@ -0,0 +1,133 @@
+package uk.gov.ons.addressIndex.parsers
+
+import com.google.common.base.CharMatcher
+
+/**
+  * A list of feature that token may take.
+  * Those feature are needed to classify the token using machine learning algorithm (Conditional Random Fields or CRF)
+  * The `private[parsers]` is needed to prevent calls from outside of the package (thus limiting it to package's
+  * classes and tests)
+  */
+object Features {
+  private val vowels = Seq('A', 'E', 'I', 'U', 'O')
+  
+  private def isAllDigits(token: String): Boolean = CharMatcher.DIGIT.countIn(token) == token.length
+
```

Comment:
Not used CharMatcher before  but I like it https://google.github.io/guava/releases/19.0/api/docs/com/google/common/base/CharMatcher.html